packages/angular-cli/models/webpack-build-common.ts

```diff
@@ -96,6 +96,7 @@ export function getWebpackCommonConfig(
         { test: /\.json$/, loader: 'json-loader' },
         { test: /\.(jpg|png|gif)$/, loader: 'url-loader?limit=10000' },
         { test: /\.html$/, loader: 'raw-loader' },
+        { test: /\.(pug|jade)/, loaders: ['apply-loader', 'pug-loader']},
```

Comment:
In the event one you'd want to add configuration here, it seems that could be done as follows:
```
       {
          test: /\.(pug|jade)$/,
          loader: 'apply',
       },
       {
          test: /\.(pug|jade)$/,
          loader: 'pug',
          query: { doctype: 'html' },
       },
```
Earlier I feared for my plugin thing I'd need to use the single loader instead (`pug-html`), but it seems that'd work fine with your setup as well.